cmd/gaia/app/export.go

```diff
@@ -63,55 +61,10 @@ func (app *GaiaApp) prepForZeroHeightGenesis(ctx sdk.Context) {
 	/* Handle fee distribution state. */
 
 	// withdraw all delegator & validator rewards
-	vdiIter := func(_ int64, valInfo distr.ValidatorDistInfo) (stop bool) {
-		err := app.distrKeeper.WithdrawValidatorRewardsAll(ctx, valInfo.OperatorAddr)
-		if err != nil {
-			panic(err)
-		}
-		return false
-	}
-	app.distrKeeper.IterateValidatorDistInfos(ctx, vdiIter)
-
-	ddiIter := func(_ int64, distInfo distr.DelegationDistInfo) (stop bool) {
-		err := app.distrKeeper.WithdrawDelegationReward(
-			ctx, distInfo.DelegatorAddr, distInfo.ValOperatorAddr)
-		if err != nil {
-			panic(err)
-		}
-		return false
-	}
-	app.distrKeeper.IterateDelegationDistInfos(ctx, ddiIter)
+	// TODO
```

Comment:
What needs to be added here? Is this necessary?

Reply:
It would probably work as is but better to clear historical rewards when exporting for zero height - updated.